evaluate_arima.py

```diff
@@ -0,0 +1,57 @@
+import warnings
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.arima_model import ARIMA
+from sklearn.metrics import mean_squared_error
+
+from get_data.get_data import call_api
+
+# Change these values to play with the ARIMA model
+n_data_points = 2000
+fraction_training = 0.9
+data = call_api(n_data_points)
+df = pd.DataFrame(data)
+df = df.drop("Date",axis=1)
+
+# evaluate an ARIMA model for a given order (p,d,q)
+def evaluate_arima_model(X, arima_order):
+	# prepare training dataset
+	train_size = int(len(X) * fraction_training)
+	train, test = X[0:train_size], X[train_size:]
+	history = [x for x in train]
+	# make predictions
+	predictions = list()
+	for t in range(len(test)):
+		model = ARIMA(history, order=arima_order)
+		model_fit = model.fit(disp=0)
+		yhat = model_fit.forecast()[0]
+		predictions.append(yhat)
+		history.append(test[t])
+	# calculate out of sample error
+	error = mean_squared_error(test, predictions)
+	return error
+
+# evaluate combinations of p, d and q values for an ARIMA model
+def evaluate_models(dataset, p_values, d_values, q_values):
+	dataset = dataset.astype('float32')
+	best_score, best_cfg = float("inf"), None
+	for p in p_values:
+		for d in d_values:
+			for q in q_values:
+				order = (p,d,q)
+				try:
+					mse = evaluate_arima_model(dataset, order)
+					if mse < best_score:
+						best_score, best_cfg = mse, order
+					print('ARIMA%s MSE=%.3f' % (order,mse))
```

Comment:
You could use f strings here, its the most up to date string formatting in python 3 and is often easier to read :) https://realpython.com/python-f-strings/